index.html

```diff
@@ -1,225 +1,240 @@
+            <ul class="tags">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="primary-btn">
+              See Project
+            </button>
           </div>
-          <div class="about-card">
-            <div class="about-card-img">
-              <img src="/images/skills.svg" alt="Skills">
-              <h2>Skills</h2>
-              <ul class="about-tags">
-                <li>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-              </ul>
-            </div>
+        </article>
+        <article class="work-card">
+          <div class="work-card-img">
           </div>
-        </div>
-      </section>
-    </main>
-    <footer class="form-section">
-      <p class="chat">
+          <div class="work-card-content">
+            <h3> 
+              Multi-Post Stories Gain+Glory 
+            </h3>
+            <ul class="tags">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="primary-btn">
+              See Project
+            </button>
+          </div>
+        </article>
+        <article class="work-card">
+          <div class="work-card-img">
+          </div>
+          <div class="work-card-content">
+            <h3> 
+              Multi-Post Stories Gain+Glory 
+            </h3>
+            <ul class="tags">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="primary-btn">
+              See Project
+            </button>
+          </div>
+        </article>
+        <article class="work-card">
+          <div class="work-card-img">
+          </div>
+          <div class="work-card-content">
+            <h3> 
+              Multi-Post Stories Gain+Glory 
+            </h3>
+            <ul class="tags">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="primary-btn">
+              See Project
+            </button>
+          </div>
+        </article>
+      </div>
+    </section>
+    <section class="about-container">
+      <div class="about-me">
+        <h2>About me</h2>
+        <p class="description">
+          I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <button type="button" class="primary-btn">
+          Get My Resume
+        </button>
+      </div>
+      <div class="about-grid">
```

Comment:
- [ ] Kindly align the cards in this section from `769px` to be 3 on 1 row, just like in the figma design, see video below to see how your website looks from my end 👍🏿 

https://user-images.githubusercontent.com/69638013/178112797-4ac05bac-d6d2-44fd-8d20-af2dad3f4cd6.mp4